lib/asn1/Makefile.am

```diff
@@ -377,132 +377,136 @@ else
 TEMPLATE_OPTION=
 endif
 
+if !CROSS_COMPILE
+$(ASN1_COMPILE_DEP): asn1_compile
+endif
+
```

Comment:
Ah, this is because `ASN1_COMPILE_DEP` expands to a relative path with `..`s in it and `make` doesn't seem to understand that it's a path to `./asn1_compile`.  This was a hack I did to make it work.  Because we use a recursive makefile setup, we could just only set `ASN1_COMPILE_DEP = asn1_compile` and use that only in `lib/asn1/Makefile.am`, and in `lib/hdb/Makefile.am` not use it at all.